### DISCUSSION TITLE
Underwater Object Simulation

### DISCUSSION THREAD

**Discussion by aagrawal05** [ProjectPhysX/FluidX3D#99] [Q&A] [answered]
Can this be used to simulate AUVs with propellers with forces of gravity and buoyancy? Also how accurate is the SPH solver to the real world?

**Comment by ProjectPhysX** [comment, marked as answer]
Yes you can do underwater simulations. Just choose the density/viscosity SI unit parameters for water accordingly in the setup script.

This is a lattice Boltzmann method solver, not SPH. Accuracy depends on a lot of factors such as resolution, suitable parametrization, geometry, boundary conditions, and also what quantity exactly you want to find out.
